### ISSUE TITLE
Using --auto-attach occupies a Terminal Window

### ISSUE THREAD

**Issue by The-Monkey-King** [dorssel/usbipd-win#556]
Running --auto-attach creates an occupying loop within a Terminal window. (As quoted by the application of it: "usbipd: info: Starting endless attach loop; press Ctrl+C to quit." Within this window now consists of messaging from the connection loop - "Attached, Detached, usbip: error: Attach Request for 1-17 failed - Device busy (exported). "

You cannot use this window without breaking the loop and thereby negating why you used it in the first place. The workaround is to open another window. 

Is it possible to have this loop running silently and free up the terminal window? Could background cmdlet be implemented in the code base (https://learn.microsoft.com/en-us/powershell/module/microsoft.powershell.core/start-job?view=powershell-7.3)?

**Comment by dorssel** [comment]
This is by design. What you really want is a background job in Linux. All `--auto-attach` does is start a script within WSL (see https://github.com/dorssel/usbipd-win/tree/master/Usbipd/wsl-scripts). You can easily run that script in the background of Linux, or perhaps even create a service for it. Remember that `usbipd wsl` commands are just convenience commands so that you don't have to run the `attach` (etc.) commands within Linux. What you are asking for is a) beyond the scope of this project, and b) easily scripted either on the Windows or on the Linux side. For the Linux side, it is of course distribution dependent, so no generic solution is available. And starting a (hidden) job in Windows is standard OS functionality. 

TL;DR: This is not going to happen any time soon, if at all.
